liquibase-core/src/main/java/liquibase/change/core/DropAllForeignKeyConstraintsChange.java

```diff
@@ -16,16 +22,14 @@
 import liquibase.structure.core.ForeignKey;
 import liquibase.structure.core.Table;
 
-import java.util.*;
-
-@DatabaseChange(name="dropAllForeignKeyConstraints", description = "Drops all foreign key constraints for a table", priority = ChangeMetaData.PRIORITY_DEFAULT, appliesTo = "table")
+@DatabaseChange(name = "dropAllForeignKeyConstraints", description = "Drops all foreign key constraints for a table", priority = ChangeMetaData.PRIORITY_DEFAULT, appliesTo = "table")
```

Comment:
Foreign keys are defined _on_ a table so this would be better as "Drops all foreign key constraints on a table".